src/metrics/http_metrics.rs

```diff
@@ -0,0 +1,72 @@
+use std::fmt::Write;
+
+use prometheus_client::encoding::{EncodeLabelSet, EncodeLabelValue, LabelValueEncoder};
+use prometheus_client::metrics::counter::Counter;
+use prometheus_client::metrics::family::Family;
+use prometheus_client::metrics::histogram::{linear_buckets, Histogram};
+use prometheus_client::registry::Registry;
+
+#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq)]
+pub enum MetricsMethod {
+    GET,
+    POST,
+    PUT,
+    DELETE,
+    OTHER,
```

Comment:
The main requests of RustyVault are GET, PUT, POST, DELETE, and LIST. Here, LIST is missing.

Reply:
Operation LIST added.